### PR TITLE
chore: updated documentation

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Updated documentation

**Related issue(s)**

- https://github.com/kyma-project/control-plane/issues/3327

### CODE REVIEW

**Review by marcobebway — APPROVED** [review]

File: metrics.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+| **kmc_edp_request_duration_seconds**                    | Duration of HTTP request to EDP in seconds.                                                                             |
+| **kmc_keb_request_duration_seconds**                    | Duration of HTTP request to KEB in seconds.                                                                             |
+| **kmc_process_sub_account_total**                       | Number of sub-accounts processed including successful and failed.                                                       |
+| **kmc_process_sub_account_processed_timestamp_seconds** | Unix timestamp (in seconds) of last successful processing of sub-account.                                               |
+| **kmc_process_old_metric_published_gauge**              | Number of consecutive re-sends of old metrics to edp per cluster. It Will reset to 0 when new metric data is published. |
+| **kmc_process_fetched_clusters**                        | All clusters fetched from KEB including trackable and not trackable.                                                    |
```
Comment (NHingerl) [inline metrics.md:10]:
```suggestion
| **kmc_process_fetched_clusters**                        | All clusters fetched from KEB, including trackable and not trackable.                                                    |
```

File: metrics.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-| **kmc_keb_number_clusters_scraped**  | Number of clusters scraped.                                         |
-| **kmc_skr_calls_total**              | Total number of calls to SKR to get the metrics of the cluster.     |
+| Metric                                                  | Description                                                                                                             |
+| ------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------- |
+| **kmc_edp_request_duration_seconds**                    | Duration of HTTP request to EDP in seconds.                                                                             |
+| **kmc_keb_request_duration_seconds**                    | Duration of HTTP request to KEB in seconds.                                                                             |
+| **kmc_process_sub_account_total**                       | Number of sub-accounts processed including successful and failed.                                                       |
```
[the remark this answers is not in the document]
Comment (mfaizanse) [reply metrics.md]: updated.

File: metrics.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+| **kmc_process_sub_account_processed_timestamp_seconds** | Unix timestamp (in seconds) of last successful processing of sub-account.                                               |
```
[the remark this answers is not in the document]
Comment (mfaizanse) [reply metrics.md]: updated.

File: metrics.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+| **kmc_process_old_metric_published_gauge**              | Number of consecutive re-sends of old metrics to edp per cluster. It Will reset to 0 when new metric data is published. |
```
[the remark this answers is not in the document]
Comment (mfaizanse) [reply metrics.md]: updated.

File: metrics.md (hunk 1 above)
Comment (mfaizanse) [reply metrics.md]: updated.

**Pull request merged**